bin/build

```diff
@@ -52,45 +46,31 @@ if [ -n "$REBUILD_RPM" ]; then
 fi
 
 pushd $IMAGE_DIR
-  cmd="$CONTAINER_COMMAND build"
-
-  # --format docker is needed for podman to ensure the builds are built with docker v2 manifests
-  if [ "$CONTAINER_COMMAND" == "podman" ]; then
-    cmd+=" --format docker"
-  fi
-
-  cmd+=" --tag $REPO/manageiq-base:$TAG \
-         --pull \
-         --build-arg RPM_PREFIX=$RPM_PREFIX"
-
-  if [ -n "$NO_CACHE" ]; then
-    cmd+=" --no-cache"
-  fi
-
-  if [ -n "$LOCAL_RPM" ] || [ -n "$REBUILD_RPM" ]; then
-    cmd+=" --build-arg LOCAL_RPM=true"
-  fi
+  # Pull the base image first
+  $CONTAINER_COMMAND pull registry.access.redhat.com/ubi9/ubi
```

Comment:
This avoids needing the `--pull` later on, which was causing the new manageiq-rpms image to need to be pushed somewhere first (so it could later be pulled)